site/docs/about.md

```diff
@@ -22,7 +22,7 @@ Iceberg is a high-performance format for huge analytic tables. Iceberg brings th
 <div class="button-box">
 <ul class="list-inline intro-social-buttons">
     <li>
-        <a href="/getting-started" class="btn btn-default btn-lg">
+        <a href="/spark-quickstart" class="btn btn-default btn-lg">
```

Comment:
Thanks for starting a PR here @danicafine. When I see a button with Learn more, I expect more docs, how about linking to the Docs introduction: https://iceberg.apache.org/docs/latest/

```suggestion
        <a href="/docs/latest/" class="btn btn-default btn-lg">
```

Thoughts?

Reply:
Oh, I now see @RussellSpitzer's comment: https://github.com/apache/iceberg/issues/12265#issuecomment-2659823166

I'm fine with both :)

Reply:
Yeah it used to go to the generic quickstart before we split it I believe. Either is also fine with me but I think I slightly prefer the quickstart.